rocksdb_replicator/replicated_db.cpp

```diff
@@ -208,6 +236,10 @@ RocksDBReplicator::ReplicatedDB::ReplicatedDB(
     replicator_helix_cluster_ = FLAGS_replicator_helix_cluster;
   }
 
+  if (FLAGS_replicator_timeout_ms > kMinReplTimeoutMs) {
+    current_replication_timeout_ms_ = FLAGS_replicator_timeout_ms;
+  }
+
```

Comment:
Why is this code snippet required? (Can you also add a comment explaining the same)?

Reply:
added a comment to explain: basically this is an extra safety check to make sure we don't set a timeout that is too-short